experimental/packages/opentelemetry-instrumentation-grpc/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "../../../tsconfig.base.json",
   "compilerOptions": {
+    "noImplicitOverride": false,
```

Comment:
This was needed as the code generated by the protobuf-ts plugin uses implicit overrides. I was able to change the config so that it adds `ts-nocheck` to the generated files. So I removed this line again :slightly_smiling_face:  [4b681f4](https://github.com/open-telemetry/opentelemetry-js/pull/3804/commits/4b681f4f911b6980de37272b9d6c6ffb71e93354)